config/routes.rb

```diff
@@ -72,9 +72,15 @@
         get "already-nominated", action: :already_nominated
       end
     end
+
+    get "/choose-how-to-continue", to: "choose_how_to_continue#new"
+    post "/choose-how-to-continue", to: "choose_how_to_continue#create"
+    get "/choice-saved", to: "choose_how_to_continue#choice_saved"
```

Comment:
Do we want to make `choose_how_to_continue` a resource?

Reply:
Normally I would but there were some issues with paths (although I can't remember what exactly) and this was the simplest way to maintain the existing`/nominations` paths